assets/js/controllers/datepicker_controller.js

```diff
@@ -0,0 +1,141 @@
+/*!
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+import { Controller } from '@hotwired/stimulus';
+import { TempusDominus, Namespace, loadLocale } from '@eonasdan/tempus-dominus';
+import { faFiveIcons } from '@eonasdan/tempus-dominus/dist/plugins/fa-five';
+
+export default class extends Controller {
+  static outlets = ['datepicker'];
+
+  static values = {
+    options: Object,
+  };
+
+  #allowedLocales = [
```

Comment:
There is way less local allowed than before, no ?

Reply:
yes, way less locales. Anyway the locale here is used mainly for texts that do not appear normally, they are hidden on long hovers.

Also localization is used to decide the format of the date, which is not super important since the date format is decided on php side with intl and transformed to javascript format.

So at the end is not a big deal.